components/InformationBoxes.jsx

```diff
@@ -30,7 +30,7 @@ const InformationBoxes = ({ boxes }) => {
         <div className={`usa-dt-information-boxes ${numberToText[boxes.length]}-boxes`}>
             {
                 boxes.map((box) => (
-                    <div key={box.title} className="usa-dt-information-box">
+                    <div key={box.type} className="usa-dt-information-box">
```

Comment:
This should be added to the PropTypes with `.isRequired`.

Reply:
Added `type: PropTypes.string.isRequired`